core/lib/network-request.js

```diff
@@ -372,16 +373,22 @@ class NetworkRequest {
     // Don't recompute times if the data is invalid. RequestTime should always be a thread timestamp.
     // If we don't have receiveHeadersEnd, we really don't have more accurate data.
     if (timing.requestTime === 0 || timing.receiveHeadersEnd === -1) return;
+
     // Take networkRequestTime and responseHeadersEndTime from timing data for better accuracy.
```

Comment:
```suggestion
    // networkRequestTime and responseHeadersEndTime are canonically defined in resource timing. Up till now, responseHeadersEndTime has a bogus fallback that was set in _onResponse().
```

Reply:
lmk if the new comment is still lacking for u